File: algorithms(dart)/insertionSort.dart
void insertionSort(List<int> numbers) {
  int temp;
  for (int i = 1; i < numbers.length; i++) {
    int j = i;
    while (j > 0) {
      if (numbers[j] < numbers[j - 1]) {
        temp = numbers[j];
        numbers[j] = numbers[j - 1];
        numbers[j - 1] = temp;
      }
      j--;
      // print(i);
    }
  }
}

File: algorithms(python)/insertion_sort.py
from typing import List

def insertionSort(numbers: List):
    for i in range(1, len(numbers)):
        while(i>0):
            if numbers[i] < numbers[i-1]:
                numbers[i], numbers[i-1] = numbers[i-1], numbers[i]
            i-=1


def insertionSortReverse(numbers: List):
    for i in range(1, len(numbers)):
        while(i>0):
            if numbers[i] > numbers[i-1]:
                numbers[i], numbers[i-1] = numbers[i-1], numbers[i]
            i-=1